docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -1,120 +1,96 @@
 
 
-## Single sign-on is coming to the B.C. government’s GitHub organizations
-
-We enabled single sign-on (SSO) for GitHub. We’re starting with the [bcgov GitHub organization](https://github.com/bcgov). After we finish transitioning [bcgov](#bcgov), we’ll enable other OCIO-managed GitHub organizations. 
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
+## Single Sign-On 
 
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to the bcgov GitHub organization, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled. 
 
 [Learn more about IDIRs](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/). 
- 
-[Learn more about the timelines and instructions](bc-government-organizations-in-github.md#the-changes). 
   
-### The changes 
- 
-Before this upgrade, users only needed their personal GitHub IDs to access the bcgov GitHub organization as a member. 
  
-By **July 30, 2024**, users must follow prompts in GitHub to link their personal GitHub ID with their IDIR account.  If not, users will be automatically removed. 
+### Directions to link your account
 
-If the instructions to re-add users doesn't work, please submit a ticket at [Jira Service Manager](https://citz-do.atlassian.net/servicedesk/customer/portal/2)
+If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
  
-[Learn more about how linking works](github-transition-guide.md#technical-information).  
-
-### Costs
+1. Once you have access again, your teammates with admin access may need to re-invite you to the repositories you had access to before.  
+2. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled.
+    - [Learn more about how to do this](github-transition-guide.md#resetting-github-keys). 
  
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
+### Upgrading bcgov-c 
+
+
+We begin transition of [`bcgov-c`](#bcgov-c) into Enterprise GitHub on **August 7, 2024**. After migration completes, we'll enable SSO for [`bcgov-c`](#bcgov-c) on **August 21, 2024**, using the same process as the SSO enablement for `bcgov`.
   
-### Directions to link your accounts before July 30
+For current B.C. Government private Enterprise organizations, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps  
```

Comment:
For current B.C. Government private organizations that are part of the legacy `bcgov-ent` Enterprise implementation, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps